src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -210,20 +210,61 @@ public void clickVisibilityOptionForResponseCommentAndSave(String idString, int
         ThreadHelper.waitFor(1000);
     }
 
-    public boolean verifyAllResultsPanelBodyVisibility(boolean visible) {
-        int numOfQns = browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse")).size();
-        assertTrue(numOfQns > 0);
-
-        // Wait for the total duration according to the number of collapse/expand intervals between questions
-        ThreadHelper.waitFor((numOfQns * 50) + 1000);
+    /**
+     * Checks if the body of all the result panels are visible
+     * @return true if all result panel body are visible
+     */
+    public boolean areResultsVisible() {
+        return isAllResultsPanelBodyVisibilityEquals(true);
+    }
 
-        for (WebElement e : browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse"))) {
-            if (e.isDisplayed() != visible) {
+    /**
+     * Checks if the body of all the result panels are hidden
+     * @return true if all result panel body are hidden
+     */
+    public boolean areResultsHidden() {
+        return isAllResultsPanelBodyVisibilityEquals(false);
+    }
```

Comment:
Be careful here. The negation of "for all x, y applies" is "there exists x such that y does not apply". Your method name and javadoc imply "for all x, y does not apply" which is wrong.